src/components/DevModePanel/useDevModeKeyTrigger.ts

```diff
@@ -1,17 +1,28 @@
 import { useEffect } from "react";
 
-type DevModeKeyTriggerCallback = () => void;
+type HandleOpen = () => void;
+type HandleClose = () => void;
 
-export const useDevModeKeyTrigger = (callbackHandler: DevModeKeyTriggerCallback) => {
+export const useDevModeKeyTrigger = (handleOpen: HandleOpen, handleClose: HandleClose) => {
   useEffect(() => {
-    const handler = (event: KeyboardEvent) => {
+    const keyDownHandler = (event: KeyboardEvent) => {
       if ((event.metaKey || event.ctrlKey) && event.code === "Quote") {
-        callbackHandler();
+        handleOpen();
       }
     };
 
-    document.addEventListener("keydown", handler);
+    const keyUpHandler = (event: KeyboardEvent) => {
+      if (event.code === "Escape") {
+        handleClose();
+      }
+    };
 
-    return () => document.removeEventListener("keydown", handler);
-  }, [callbackHandler]);
+    document.addEventListener("keydown", keyDownHandler);
+    document.addEventListener("keyup", keyUpHandler);
+
+    return () => {
+      document.removeEventListener("keydown", keyDownHandler);
+      document.removeEventListener("keyup", keyUpHandler);
```

Comment:
question: what is the reason to use separate triggers to open and close, IMO it could be the same. I know that in most cases no one notice that close fires when you release the esc key but for some people it might be confusing.

Reply:
@poulch surprisingly `keydown` event is not fired for Escape key because the playground stops propagation of this event. I'll add a comment